.github/workflows/updater-check.yaml

```diff
@@ -0,0 +1,60 @@
+name: Start Updater
+on:
+  push:
+    branches:
+    - 'yli3/**'
```

Comment:
Please remove the branch trigger before merging :)

Reply:
Should I replace with this?
```
on:
  schedule:
    - cron: '0 */3 * * *'
  push:
    branches:
      - master
```

Reply:
That would trigger your pipeline on each push to master. Is that necessary?

I would think that you'd rather want to look for new tags. 

```
on:
  schedule: 
    - cron: '0 */3 * * *'
  push:
    tags: 
      - '*'
```